tests/migrations/test_writer.py

```diff
@@ -647,3 +647,7 @@ def deconstruct(self):
 
         string = MigrationWriter.serialize(models.CharField(default=DeconstructibleInstances))[0]
         self.assertEqual(string, "models.CharField(default=migrations.test_writer.DeconstructibleInstances)")
+
+    def test_serialize_type_none(self):
+        # Regression test for issue #29814.
```

Comment:
We usually only keep reference to ticket in tests when it's hard to express the complexity of the issue in Python. In this case the issue and resolution is pretty trivial so you can drop it entirely. `git blame` and the commit message should do a good job at pointing at the ticket if it's required in the future.

Reply:
Okay, makes sense. Do you think I should remove it for this pull request or just keep that in mind for the next time?

Reply:
remove it from here.

Reply:
Alright!